### DISCUSSION TITLE
How to make useFieldArray inside useFieldArray?

### DISCUSSION THREAD

**Discussion by difaananda40** [react-hook-form/react-hook-form#2923] [General] [answered]
Hello, like the title above, I want to implement fieldArray inside fieldArray, is that possible?

**Comment by bluebill1049** [comment, marked as answer]
there is an example in the examples folder: https://codesandbox.io/s/react-hook-form-usefieldarray-nested-arrays-x7btr

**Reply by difaananda40** [reply]
ah, i didn't see that before, thanks!

**Comment by prakash03dev** [comment]
Can you add typescript?
